src/pudl_archiver/archivers/epacems.py

```diff
@@ -15,6 +20,38 @@
 logger = logging.getLogger(f"catalystcoop.{__name__}")
 
 
+class BulkFile(BaseModel):
```

Comment:
Wasn't *necessary* for this change, but found it slightly helpful to document the expected shape of the API responses in code - and made the type annotations more useful.